packages/cozy-harvest-lib/src/index.js

```diff
@@ -1,39 +1,10 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import I18n from 'cozy-ui/react/I18n'
-
-import TranslatedDeleteAccountButton from './components/DeleteAccountButton'
-import TranslatedTriggerManager from './components/TriggerManager'
-import TranslatedTriggerLauncher from './components/TriggerLauncher'
+export {
+  default as DeleteAccountButton
+} from './components/DeleteAccountButton'
+export {
+  default as withKonnectorModal
+} from './components/hoc/withKonnectorModal'
```

Comment:
`export withKonnectorModal from './components/hoc/withKonnectorModal'` doesn't work here?

Reply:
When using this syntax, I got `Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled` in Cozy-Bar :sob: